Pulsarc/Skinning/Skin.cs

```diff
@@ -238,7 +256,75 @@ static public float getConfigFloat(string config, string section, string key)
         /// <returns>The Anchor found using the provided parameters.</returns>
         static public Anchor getConfigAnchor(string config, string section, string key)
         {
-            return (Anchor) Enum.Parse(Anchor.TopLeft.GetType(),configs[config][section][key]);
+            return (Anchor)Enum.Parse(Anchor.TopLeft.GetType(), getConfigString(config, section, key));
+        }
+
+        /// <summary>
+        /// Find the config provided, go to the section provided, and return the start position of the
+        /// key provided. If parent is not null, the start position will be based on the parent.
+        /// If parent is null, start position will be based on the screen.
+        /// </summary>
+        /// <param name="config">The config to look in.</param>
+        /// <param name="section">The section of a config to look in.</param>
+        /// <param name="key">The name of the variable.</param>
+        /// <returns>
+        /// The position found using the provided parameters.
+        /// Currently only ScreenAnchor and pre-determined parent position finding is supported.
+        /// </returns>
+        static public Vector2 getConfigStartPosition(string config, string section, string key, Drawable parent = null)
+        {
+            string anchorString = getConfigString(config, section, key);
+
+            if (parent == null)
+            {
+                return AnchorUtil.FindScreenPosition((Anchor)Enum.Parse(Anchor.TopLeft.GetType(), anchorString));
+            }
+            else
```

Comment:
Unnecessary else statement.

Also technically you can do this to return instead:
```csharp
return parent == null
    ? AnchorUtil.FindScreenPosition((Anchor)Enum.Parse(Anchor.TopLeft.GetType(), anchorString))
    : AnchorUtil.FindDrawablePosition((Anchor)Enum.Parse(Anchor.TopLeft.GetType(), anchorString), parent);
```

Reply:
Originally this method was a lot more complicated for no real reason, the if/else statement is basically the skeleton that remained after I removed all that extra stuff. I've implemented your suggestion, thanks!